doc/release-notes.md

```diff
@@ -20,6 +20,11 @@ shut down (which might take a few minutes for older versions), then run the
 installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
 bitcoind/bitcoin-qt (on Linux).
 
+**Mac Upgrader Notice:** Bitcoin-Core 0.11 uses a new application bundle name on osx
+(renamed to Bitcoin-Core from Bitcoin-Qt). You should therefore delete
+/Applications/Bitcoin-Qt after copy over of the new /Applications/Bitcoin-Core
+application bundle.
+
```

Comment:
Hmm... right. It is not ideal.
But either we rename the app now, with the consequence, that the user needs to delete the old app (now we are in version 0.11 which implies that such things could be necessary) or we keep the name Bitcoin-Qt.app as application bundle name.
I have to admit that it is inconsistent at the moment (OSX: "Bitcoin-Core", win/linux: "bitcoin-qt" [name of the runnable application from the enduser perspective]).